package.json

```diff
@@ -11,7 +11,8 @@
     "build": "docusaurus build",
```

Comment:
Why no longer build the API doc on build but commit to source code?

Reply:
So that this feature here works: 
![image](https://user-images.githubusercontent.com/27026029/191218557-541280ba-0bfb-47ef-96b2-b6b4c378f99c.png)